Tutorial_1c_Line_Ratios.ipynb

```diff
@@ -0,0 +1,736 @@
+{
```

Comment:
<p>Can also link here to the radio-astro-tools tutorial on this here: <a href="https://github.com/radio-astro-tools/tutorials/blob/master/SpectralCubeReprojectExample.ipynb" target="_blank">https://github.com/radio-astro-tools/tutorials/blob/master/SpectralCubeReprojectExample.ipynb</a></p><p> </p><p>Specifically, it handles the case where the spectral channels are different widths and how to match those.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>